bobtemplates/django/+package.fullname+/+package.fullname+/api/tests/test_api.py

```diff
@@ -0,0 +1,2 @@
+def test_api_tests_are_run():
+    assert True
```

Comment:
I'm not sure about the value of this test: it would only bring value if it would fail when it is not run, but chance for that is pretty low 😉😎
But it may be a good place to start implementing a better test in the project...

Is there any thing we could actually assert? Whether the API-stuff is available, for example?

Reply:
I was just wondering if the tests were executed.